apps/rush-lib/src/schemas/command-line.schema.json

```diff
@@ -187,6 +187,28 @@
               "items": {
                 "type": "string"
               }
+            },
+            "watchOptions": {
+              "title": "Watch Options",
+              "description": "Controls the file watching behavior of this command. If not specified, this command does not watch files.",
+              "type": "object",
+              "additionalProperties": false,
+              "required": ["watchForChanges", "watchPhases"],
+              "properties": {
+                "watchForChanges": {
+                  "title": "Watch For Changes",
+                  "description": "Indicates that this command will watch for changes after the initial execution.",
+                  "enum": [true]
```

Comment:
Does this implicitly make the type `boolean`? Should we allow someone to say `false` for whatever reason?

Maybe this should be a boolean in the schema and if they say `false`, we throw a more meaningful error message than "this doesn't match the schema?"

Reply:
I'd rather drop the property altogether, since is it redundant. The presence of `watchOptions` makes it a watch action (what's the point of defining `watchOptions` for an action that doesn't watch?). Making it only accept `true` was a compromise with the request to include the property.

Reply:
Switched to the `alwaysWatch` option, which, when `false` allows a `--watch` CLI flag to enable the watcher (provided that `watchPhases` is non-empty).